### ISSUE TITLE
Crash when expanding the timeline - Zoom in

### ISSUE THREAD

**Issue by steveking16** [smeighan/xLights#2323]
I've had xLights crash on me 4 times over the last 3 days.  In each case, it was when I clicked the zoom button to expand the timeline.  Relatively early in the sequence edit cycle.  All on imported effects sequences.  A couple before I had a chance to save (but early enough I didn't lose much).  

Happy to send any additional info you may need.

[xLights_dbgrpt-250940-20201104T134213.zip](https://github.com/smeighan/xLights/files/5498070/xLights_dbgrpt-250940-20201104T134213.zip)
[xLights_dbgrpt-283076-20201104T095453.zip](https://github.com/smeighan/xLights/files/5498071/xLights_dbgrpt-283076-20201104T095453.zip)
[xLights_dbgrpt-294456-20201105T154142.zip](https://github.com/smeighan/xLights/files/5498072/xLights_dbgrpt-294456-20201105T154142.zip)
[xLights_dbgrpt-239372-20201103T122602.zip](https://github.com/smeighan/xLights/files/5498073/xLights_dbgrpt-239372-20201103T122602.zip)

**Comment by keithsw1111** [comment]
I would like to see a video of where and how you are zooming in. The crash definitely relates to the zoom and reading the audio data ... but it this was common we would see more crashes like this ... and we dont. So it is either something about your audio file or something about how you are doing the zoom in.

**Comment by steveking16** [comment]
I just clicked the zoom button (the plus magnifier icon). I can click it after the restart without a problem. I’ve been trying to identify any consistent conditions but can’t help.

Developer background so understand your (and my) frustration

-Steve

Please excuse brevity and mistakes as this was sent from my iPhone

On Nov 5, 2020, at 6:26 PM, Keith Westley <notifications@github.com> wrote:

﻿

I would like to see a video of where and how you are zooming in. The crash definitely relates to the zoom and reading the audio data ... but it this was common we would see more crashes like this ... and we dont. So it is either something about your audio file or something about how you are doing the zoom in.

**Comment by keithsw1111** [comment]
Right ... which is why we need to SEE what you are doing and the state the program is in. Ideally going from working to not working but even just not working will help.

**Comment by steveking16** [comment]
Understood. Not sure what I need to do. Inconsistent about when it occurs. All four cases are different sequences. I can’t predict when it will occur — so can’t record it. Would not have reported until it happened four times.

Please excuse brevity and mistakes as this was sent from my iPhone

On Nov 5, 2020, at 6:33 PM, Keith Westley <notifications@github.com> wrote:

﻿

Right ... which is why we need to SEE what you are doing and the state the program is in. Ideally going from working to not working but even just not working will help.

**Comment by keithsw1111** [comment]
So that makes it difficult. Where the focus was(in the timeline) may be a key factor in why one crashes and one doesnt. Were you playing the song when it crashed?

**Comment by steveking16** [comment]
I understand how difficult this is.  I was reluctant to post the first couple because they seemed so sporadic and arbitrary.

I don’t think I was actually playing at the time.  I think I had just found a place where I needed to adjust the layer settings on a group, so stopped the song at that point and was trying to expand the region to select the effect needing adjustment – but wasn’t paying that much attention prior to the crash.

This is helpful for me though to know what information is more likely to help you – like was the audio playing or not.  I hope it doesn’t happen again, but will try to note what I’m doing more closely.

Steven King
steveking16@outlook.com<mailto:steveking16@outlook.com>
(307) 630-3769 mobile
